src/riak_kv_coverage_filter.erl

```diff
@@ -150,26 +154,26 @@ build_item_filter(FilterInput) ->
 
 
 %% @private
-build_preflist_fun(Bucket, CHBin) ->
+build_preflist_fun(Bucket, CHBin, KeyConvFn) ->
     fun({o, Key, _Value}) -> %% $ index return_body
-            ChashKey = riak_core_util:chash_key({Bucket, Key}),
+            ChashKey = riak_core_util:chash_key({Bucket, KeyConvFn(Key)}),
             chashbin:responsible_index(ChashKey, CHBin);
        ({_Value, Key}) ->
-            ChashKey = riak_core_util:chash_key({Bucket, Key}),
+            ChashKey = riak_core_util:chash_key({Bucket, KeyConvFn(Key)}),
```

Comment:
Is it necessary to apply the identity function on non-TS keys? The `KeyConvFn` here can only be the dumb `fun(X) -> X end`, can it?